packages/core/src/base/body.scss

```diff
@@ -37,5 +37,15 @@ Style guide: style.base
   @extend %ds-base;
   background-color: $color-background-inverse;
   color: $color-base-inverse;
```

Comment:
I needed to add styles for links that are used in a `.ds-base--inverse` section. 
I thought this was the best place to add them unless folks think these styles should be in with the `typography` styles